backend/apps/rollback_migration.py

```
@@ -0,0 +1,27 @@
+        if options['migration']:
+            self.stdout.write(f'📋 Rolling back {options["migration"]}')
+        
+        with connection.cursor() as cursor:
+            # Get last migration
+            cursor.execute("SELECT app, name FROM django_migrations ORDER BY id DESC LIMIT 1")
+            last = cursor.fetchone()
+            
+            if last:
+                self.stdout.write(f'✅ Rolling back: {last[0]}.{last[1]}')
+                # Simulate rollback
+                cursor.execute(f"DELETE FROM django_migrations WHERE app='{last[0]}' AND name='{last[1]}'")
```

Ruff (0.15.21)

[error] Possible SQL injection vector through string-based query construction

(S608)